host_os.py

```diff
@@ -31,12 +35,41 @@
     'set-env': setup_environment,
     'build-iso': build_iso,
 }
+REQUIRED_PACKAGE_DEPENDENCIES = [
+    "bzip2", "git", "GitPython", "lzop", "mock", "nosync", "python-lxml", "PyYAML", "subversion", "wget"
+]
+
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -qa | grep %s" % package_name
```

Comment:
I think `"rpm -q %s" % package_name` ought to suffice. Then we could cleanup the logic below and just check the return code.